generators/app/templates/src/containers/TicTacToe/GameListPage/children/List/list.component.js

```diff
@@ -187,10 +187,28 @@ const List = ({ webId, gamePath, sendNotification }: Props) => {
   const getGames = useCallback(
     async url => {
       try {
+        let gameItemPredicate;
         const document = await ldflexHelper.fetchLdflexDocument(url);
         let gameList = [];
         if (!document) return gameList;
-        for await (const item of document['schema:hasPart']) {
+
+        const type = await document['rdf:type'];
+        const typeValue = type ? type.value : undefined;
+
+        /**
+         * If the document is a container, then we want to loop over the ldp:contains property of the container
+         * If it is not a container, we are using schema:hasPart so we don't confuse it with a container
+         * schema:hasPart is used for externally linked games in other people's pods
+         */
+        if (typeValue && typeValue === LDPVocabulary.BASICCONTAINER) {
```

Comment:
The `===` should be enough. No need to use `&&` to check if `typeValue` is falsy.